packages/dso-toolkit/src/components/logo/logo.models.ts

```diff
@@ -0,0 +1,10 @@
+export interface Logo {
+  logo: string;
+  subLogo: string;
+  showSubLogo: boolean;
+  beta: boolean;
+}
```

Comment:
Ik heb de scope van het issue bijgewerkt. De template models vertalen het visuele ontwerp tot een technische implementatie.

Ik denk dat we aan het volgende template model genoeg hebben:

```suggestion
export interface Logo {
  label?: string;
  ribbon?: string;
}
```